web/app.js

```diff
@@ -2807,6 +2808,7 @@ function webViewerKeyDown(evt) {
               source: window,
               type: "again",
               findPrevious: cmd === 5 || cmd === 12,
+              matchDiacritics: state.matchDiacritics,
```

Comment:
This line shouldn't be necessary now, right?
I'm guessing that this might be fallout from rebasing on top of other changes.